org.mixxx.Mixxx.yaml

```diff
@@ -44,175 +56,290 @@ cleanup:
     sources:
       - type: archive
-        url: https://github.com/PortMidi/portmidi/archive/refs/tags/v2.0.3.tar.gz
-        sha256: 934f80e1b09762664d995e7ab5a9932033bc70639e8ceabead817183a54c60d0
+        url: https://github.com/PortMidi/portmidi/archive/refs/tags/v2.0.4.tar.gz
+        sha256: 64893e823ae146cabd3ad7f9a9a9c5332746abe7847c557b99b2577afa8a607c
     cleanup:
       - /bin
 
+  # USB and Bluetooth HID library
   - name: hidapi
     buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
     sources:
       - type: archive
-        url: https://github.com/libusb/hidapi/archive/refs/tags/hidapi-0.12.0.tar.gz
-        sha256: 28ec1451f0527ad40c1a4c92547966ffef96813528c8b184a665f03ecbb508bc
+        url: https://github.com/libusb/hidapi/archive/refs/tags/hidapi-0.14.0.tar.gz
+        sha256: a5714234abe6e1f53647dd8cba7d69f65f71c558b7896ed218864ffcf405bcbd
 
+  # Audio metadata library
   - name: taglib
     buildsystem: cmake-ninja
     config-opts:
+      # Enable shared libraries and disable tests
       - -DBUILD_SHARED_LIBS=ON
-      - -DWITH_MP4=ON
-      - -DWITH_ASF=ON
+      - -DBUILD_TESTING=OFF
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
     sources:
       - type: archive
-        url: https://taglib.org/releases/taglib-1.12.tar.gz
-        sha256: 7fccd07669a523b07a15bd24c8da1bbb92206cb19e9366c3692af3d79253b703
+        url: https://taglib.org/releases/taglib-1.13.1.tar.gz
+        sha256: c8da2b10f1bfec2cd7dbfcd33f4a2338db0765d851a50583d410bacf055cfd0b
     cleanup:
       - /bin
 
+  # Audio fingerprinting library
   - name: chromaprint
     buildsystem: cmake-ninja
     config-opts:
+      # Disable command line tools and tests
+      - -DBUILD_TOOLS=OFF
+      - -DBUILD_TESTS=OFF
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-      - -DBUILD_SHARED_LIBS=ON
     sources:
       - type: archive
         url: https://github.com/acoustid/chromaprint/releases/download/v1.5.1/chromaprint-1.5.1.tar.gz
         sha256: a1aad8fa3b8b18b78d3755b3767faff9abb67242e01b478ec9a64e190f335e1c
 
+  # Audio time stretching and pitch shifting library
   - name: rubberband
     buildsystem: meson
+    config-opts:
+      # Disable tests, command line tools and audio plugins
+      - -Dtests=disabled
+      - -Dcmdline=disabled
+      - -Dladspa=disabled
+      - -Dlv2=disabled
     sources:
       - type: archive
-        url: https://github.com/breakfastquay/rubberband/archive/refs/tags/v2.0.2.tar.gz
-        sha256: 1372a0d52915ef07aa3cffc00a8b17996c736f328efa0f6504d231fa6428ed4b
+        url: https://github.com/breakfastquay/rubberband/archive/refs/tags/v3.3.0.tar.gz
+        sha256: 2bb837fe00932442ca90e185af8a468f7591df0c002b4a9e27a1bced1563ac84
 
+  # Audio processing Library
   - name: soundtouch
     buildsystem: cmake-ninja
+    config-opts:
+      # Disable soundstretch utility and Neon SIMD instructions
+      - -DSOUNDSTRETCH=OFF
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DNEON=OFF
```

Comment:
Why disable NEON? There's an aarch64 build.

Reply:
Interesting... here's what [SoundTouch's build system does](https://codeberg.org/soundtouch/soundtouch/src/commit/74514f559766cd24fc87aa278a5c8722c028b8ae/CMakeLists.txt#L62):

```cmake
if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(armv7.*|armv8.*|aarch64.*)$")
  set(NEON_CPU ON)
else()
  set(NEON_CPU OFF)
endif()

option(NEON "Use ARM Neon SIMD instructions if in ARM CPU" ON)
if(${NEON} AND ${NEON_CPU})
  target_compile_definitions(SoundTouch PRIVATE SOUNDTOUCH_USE_NEON)
  if(NOT CMAKE_SYSTEM_PROCESSOR MATCHES "^aarch64.*$")
    target_compile_options(SoundTouch PRIVATE -mfpu=neon)
  endif()
endif()
```

So I guess CMAKE_SYSTEM_PROCESSOR is matching `armv7` or `armv8` on Buildbot? I thought Flathub builds for aarch64 though... :thinking:  @hfiguiere do you have any suggestions on this?